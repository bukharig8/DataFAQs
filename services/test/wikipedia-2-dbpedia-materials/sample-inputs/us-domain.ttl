@prefix gn: <http://www.geonames.org/ontology#> .
<http://en.wikipedia.org/wiki/.us> a gn:WikipediaArticle .
