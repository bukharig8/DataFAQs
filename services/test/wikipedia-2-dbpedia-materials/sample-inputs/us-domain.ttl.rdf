<?xml version="1.0" encoding="utf-8"?>
<rdf:RDF xmlns:gn="http://www.geonames.org/ontology#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#">
  <rdf:Description rdf:about="http://en.wikipedia.org/wiki/.us">
    <rdf:type rdf:resource="http://www.geonames.org/ontology#WikipediaArticle"/>
  </rdf:Description>
</rdf:RDF>
