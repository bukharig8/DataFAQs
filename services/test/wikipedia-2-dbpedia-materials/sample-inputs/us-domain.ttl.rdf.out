<rdf:RDF
    xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
    xmlns:j.0="http://www.w3.org/ns/prov#">
  <j.0:Entity rdf:about="http://en.wikipedia.org/wiki/.us"/>
</rdf:RDF>
